packages/app/src/models/history.ts

```diff
@@ -56,11 +54,13 @@ export const wipe = () => {
 
 /** add a line of repl history */
 export const add = (line: HistoryLine) => {
-  if (lines.length === 0 || JSON.stringify(lines[lines.length - 1]) !== JSON.stringify(line)) {
-    // don't add sequential duplicates
-    lines.push(line)
-    store().setItem(key, JSON.stringify(lines))
-    // console.log('history::add', cursor)
+  if (line.raw !== 'history -c' && line.raw !== 'history --clear') { // don't add history clear command
```

Comment:
is this a new decision, to redact all `clear` executions from command history? if we want this, then it should be specified in the command registration, rather than here in the history model?

Reply:
Thanks! As we discussed offline, I updated my PR with a new Entity interface IRedact, which means the should not be stored in the history and returns nothing.